src/Psalm/Internal/Analyzer/ClosureAnalyzer.php

```diff
@@ -51,10 +51,12 @@ public function getTemplateTypeMap(): ?array
      */
     public function getClosureId(): string
     {
-        return strtolower($this->getFilePath())
-            . ':' . $this->function->getLine()
-            . ':' . (int)$this->function->getAttribute('startFilePos')
-            . ':-:closure';
+        return strtolower(
+            $this->getFilePath()
+                . ':' . $this->function->getLine()
+                . ':' . (int)$this->function->getAttribute('startFilePos')
+                . ':-:closure'
+        );
```

Comment:
Just realized this should be fixed instead of changing here. Concatenating int should still result in lowercase string.